internal/pkg/cli/progress.go

```diff
@@ -27,11 +27,11 @@ var defaultResourceCounts = map[termprogress.Text]int{
 
 // Row descriptions displayed while deploying an environment.
 const (
-	textVPC             termprogress.Text = "- Virtual private cloud on 2 availability zones to hold your services"
+	textVPC             termprogress.Text = "- Virtual private cloud on 2 availability zones to hold your services and jobs"
 	textInternetGateway termprogress.Text = "  - Internet gateway to connect the network to the internet"
-	textPublicSubnets   termprogress.Text = "  - Public subnets for internet facing services "
-	textPrivateSubnets  termprogress.Text = "  - Private subnets for services that can't be reached from the internet"
-	textRouteTables     termprogress.Text = "  - Routing tables for services to talk with each other"
```

Comment:
Were the end-of-line extra spaces in `textPublicSubnets` and `textECSCluster` intentional? If so I can add them back in.

Reply:
Nope they weren't :) so it should be good!